pull_request_template.md

```diff
@@ -1,8 +1,8 @@
-<!-- Please fill in the **bold** fields and tick all applicable boxes in the "Preview" tab. -->
+<!-- Please fill in the **bold** fields and tick all applicable boxes by placing an "x" inside "[ ]". -->
```

Comment:
I have doublechecked it on Safari (OS X): it does not work. Maybe if the post has already been made that the preview tab works?

E.g. when writing a message
```
 - [x] checked
 - [ ] unchecked
```

looks like
<img width="146" alt="screen shot 2016-06-01 at 16 08 09" src="https://cloud.githubusercontent.com/assets/177360/15712583/2a2dfab2-2813-11e6-8da8-5835e72ccb25.png">
in the Preview tab. Note that both are greyed out, i.e. not interactive.